deployment/ccip/changeset/testhelpers/test_helpers.go

```diff
@@ -482,6 +482,45 @@ func AddLane(
 	require.NoError(t, err)
 }
 
+func UpdateLane(
+	t *testing.T,
+	e *DeployedEnv,
+	from, to uint64,
+	isTestRouter, isEnabled bool,
+) {
+	var err error
+	var apps []commoncs.ChangesetApplication
+	apps = append(apps, commoncs.ChangesetApplication{
```

Comment:
To disable a lane, I think we need to disable the router to ramp connection, otherwise router will still route the requests to ramps